### PR TITLE
I added comment

### PR DESCRIPTION
**Pull request by GokceKochan**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,6 +3,14 @@
...
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <dependencies>
+        <dependency>
+            <groupId>io.cucumber</groupId>
+            <artifactId>messages</artifactId>
+            <version>17.1.1</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
```
Comment (ademcigdem) [inline pom.xml:13]: Thanks for that

**Review by ademcigdem — APPROVED** [review]

**Comment by ecaliskanunal** [comment]
You could use // to add a single line comment.

Love,
Elif

**Comment by Zorlu-Test** [comment]
Ok

**Comment by Zorlu-Test** [comment]
done

File: pom.xml [hunk 2]
```diff
@@ -3,6 +3,14 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <dependencies>
+        <dependency>
+            <groupId>io.cucumber</groupId>
+            <artifactId>messages</artifactId>
+            <version>17.1.1</version>
```
Comment (Zorlu-Test) [inline pom.xml:10]: Please delete this row

**Review by fikretellek — APPROVED** [review]

**Review by Zorlu-Test — APPROVED** [review]
Please fix issue

**Comment by GokceKochan** [comment]
done

**Pull request merged**